### PR TITLE
Update UserGuide.md, README.md, AboutUs.md

### PR DESCRIPTION
**Pull request by j4ck990**

### CODE REVIEW

**Review by Vishwanth2210 — CHANGES_REQUESTED** [review]
Image Loading Issue (ad-nap.png) is still present. Will check back on 22 Feb to allow GitHub to load the image if necessary. Other than that, it looks good for merging.

File: docs/AboutUs.md [hunk 1]
```diff
@@ -13,29 +13,30 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
 
 * Role: Project Advisor
 
 ### Mohd Shafik
 
 <img src="images/ad-nap.png" width="200px">
 
-[github](https://github.com/AD-NAP/)
-[portfolio](team/ad-nap.md)
+[[github](https://github.com/AD-NAP/)]
```
Comment (AD-NAP) [inline docs/AboutUs.md:26]:
```suggestion
[github](https://github.com/AD-NAP/)
```

File: docs/AboutUs.md [hunk 2]
```diff
@@ -13,29 +13,30 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
+[[portfolio](team/ad-nap.md)]
```
Comment (AD-NAP) [inline docs/AboutUs.md:27]:
```suggestion
[portfolio](team/ad-nap.md)
```

File: docs/AboutUs.md [hunk 3]
```diff
@@ -13,29 +13,30 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
 * Responsibilities: UI
 
-### Johnny Doe
+### Jack Ng
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/j4ck990.png" width="200px">
 
-[[github](http://github.com/johndoe)] [[portfolio](team/johndoe.md)]
+[[github](http://github.com/j4ck990)]
```
Comment (AD-NAP) [inline docs/AboutUs.md:36]:
```suggestion
[github](http://github.com/j4ck990)
```

File: docs/AboutUs.md [hunk 4]
```diff
@@ -13,29 +13,30 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
+[[portfolio](team/j4ck990.md)]
```
Comment (AD-NAP) [inline docs/AboutUs.md:37]:
```suggestion
[portfolio](team/j4ck990.md)
```

**Review by AD-NAP — CHANGES_REQUESTED** [review]
I faced the same issue where hyperlinks []() format becomes enclosed by another set of []. Not sure what's causing this tho

File: docs/AboutUs.md (hunk 1 above)
Comment (j4ck990) [reply docs/AboutUs.md:26]: If i not wrong thats just so the hyper link is wrapped by a set of [] when viewing the markdown. 
I tested it does not display properly on IntelliJ but does work on Github

Comment (j4ck990) [reply docs/AboutUs.md:26]: If I am not wrong that is just so the hyper link is wrapped by a set of [] when viewing the markdown on GitHub for readability. 
I tested it does not display properly on IntelliJ but does work on GitHub.

Comment (AD-NAP) [reply docs/AboutUs.md:26]: Right, I just noticed it now. I think we can leave the [] enclosing like the original file. We'll clarify with TA next time 😄

**Comment by AD-NAP** [comment]
LGTM

**Pull request merged**